travis/buildViaTravis.sh

```diff
@@ -6,14 +6,14 @@ set -e
 
 if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
   echo -e "Build Pull Request #$TRAVIS_PULL_REQUEST => Branch [$TRAVIS_BRANCH]"
-  ./gradlew --no-daemon build javadoc asciidoc coveralls dockerBuildAllImages
+  # Build and run all tests, create coverage report
+  ./gradlew --no-daemon build coveralls
```

Comment:
So do we run the integration tests here with the in memory database and then two more times with the other databases? Why leave coveralls here but move the others?

Reply:
My logic was to move the two artifact-producing steps (asciidoc, docker) past the test, so they run (and publish) only if all tests passed.

Not being sure of how coverall works, i left it where it was (together with `build`). I'm going to try to move it after to see if we get additional coverage stats from the 2 sets of integration tests.

Reply:
I doubt the coveralls would change as the code paths should be the same but you can test it

Reply:
I agree it may not go up (we don't have SQL-specific code paths).
But it may go *down*, depending on how coverall works, because it may only publish results from the last execution (which would be integration, if i move this away from `build`), that is the reason I left it where it was. But going to try, maybe it is able to look at everything regardless of when it executed.